### PR TITLE
Pipe through app folder selection to init dataconnect:sdk

### PR DESCRIPTION
**Pull request by joehan**
### Description
Stop asking for app folder twice - just pipe through the selection made in the VSCode UI

### CODE REVIEW

File: src/init/features/dataconnect/sdk.ts [hunk 1]
```diff
@@ -75,11 +76,13 @@ async function askQuestions(setup: Setup, config: Config): Promise<SDKInfo> {
   } else {
     // If we aren't, ask the user where their app is, and try to autodetect from there
     logBullet(`Couldn't automatically detect your app directory.`);
-    appDir = await promptForDirectory({
-      config,
-      message:
-        "Where is your app directory? Leave blank to set up a generated SDK in your current directory.",
-    });
+    appDir =
```
Comment (fredzqm) [inline src/init/features/dataconnect/sdk.ts:79]: Should this happen before line 71?

If `FDC_APP_FOLDER` is specified, but `process.cwd()` is detected to be a iOS app, it should still respect `FDC_APP_FOLDER`.

**Review by fredzqm — APPROVED** [review]
Thanks! 

LG after one suggestion.

File: src/init/features/dataconnect/sdk.ts (hunk 1 above)
Comment (joehan) [reply src/init/features/dataconnect/sdk.ts:78]: Good call, fixed.

File: firebase-vscode/src/data-connect/terminal.ts [hunk 2]
```diff
@@ -11,16 +11,16 @@ const executionOptions: vscode.ShellExecutionOptions = {
...
 export function setTerminalEnvVars(envVar: string, value: string) {
   environmentVariables[envVar] = value;
 }
 
-export function runCommand(command: string) {
+export function runCommand(command: string, cwd?: string) {
+  const terminalOptions: TerminalOptions = {
+    name: "Data Connect Terminal",
+    env: environmentVariables,
+    cwd,
```
[the remark this answers is not in the document]
Comment (joehan) [reply firebase-vscode/src/data-connect/terminal.ts]: Nope, cleaned up cwd. I originally thought we could just run the command from the chosen directory, but realized it wouldn't add to the right firebase.json if the chosen dir was outside of the current firebase directory.

**Pull request merged**